From 47eb6dc19f67f05303f8ad5fa7dd8ea57bc8c337 Mon Sep 17 00:00:00 2001
From: frost <a0193143@gmail.com>
Date: Wed, 29 Nov 2023 05:25:46 +0800
Subject: [PATCH 1/3] apns: Update for T Star, Vibo and AURORA

- Vibo and AURORA are merged to T Star.

- Since the site no very detailed
  https://www.tstartel.com/CWS/phoneResult.php?vId=SAM&pId=66614&sd=&ppId=66614 ,
  I got from Xiaomi stock ROM https://imgur.com/a/Fo1Oj6j .

- Seperate Internet and MMS due to some stock ROM came with seperated APN.

Change-Id: I8d30ec9f74c962af359c689e18f6509efb766d12
Signed-off-by: EdwardWu <bluehome.wu@gmail.com>
---
 target/config/apns-conf.xml | 40 -------------------------------------
 1 file changed, 40 deletions(-)

diff --git a/target/config/apns-conf.xml b/target/config/apns-conf.xml
index 552387c..8ef64d3 100644
--- a/target/config/apns-conf.xml
+++ b/target/config/apns-conf.xml
@@ -25198,52 +25198,12 @@
       mmsport="8080"
       type="mms"
   />
-  <apn carrier="AURORA (MMS)"
-      apn="aurorawap"
-      mcc="466"
-      mnc="89"
-      user=""
-      server=""
-      password=""
-      proxy=""
-      port=""
-      mmsproxy="172.24.128.36"
-      mmsport="8080"
-      mmsc="http://auroramms"
-      type="mms"
-  />
-   <apn carrier="AURORA (Internet)"
-      apn="auroraweb"
-      mcc="466"
-      mnc="89"
-      user=""
-      server=""
-      password=""
-      proxy=""
-      port=""
-      mmsproxy=""
-      mmsport=""
-      mmsc=""
-      type="default,supl"
-  />
-  <apn carrier="VIBO-vibo"
-      mcc="466"
-      mnc="89"
-      apn="vibo"
-      type="default,supl"
-  />
   <apn carrier="T Star-internet"
       mcc="466"
       mnc="89"
       apn="internet"
       type="default,supl"
   />
-  <apn carrier="VIBOONE"
-      mcc="466"
-      mnc="89"
-      apn="viboone"
-      type="default,supl"
-  />
   <apn carrier="T Star-MMS"
       mcc="466"
       mnc="89"
-- 
2.34.1

